docs/content/latest/yugabyte-cloud/cloud-clusters/pause-clusters.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Pause, resume, and delete clusters
+linkTitle: Pause, resume, delete
+description: Pause, resume, and delete clusters in Yugabyte Cloud.
+headcontent:
+image: /images/section_icons/manage/backup.png
+menu:
+  latest:
+    identifier: pause-clusters
+    parent: cloud-clusters
+    weight: 500
+isTocNested: true
+showAsideToc: true
+---
+
+## Pause clusters
+
+To reduce costs on temporarily unused clusters, you can pause a cluster for up to 30 days. Paused clusters are not billed for instance vCPU capacity. Disk and Backup storage are charged at the standard rate. Refer to [Cluster costs](../../cloud-admin/cloud-billing-costs/#paused-cluster-costs).
+
+To pause a cluster, on the **Clusters** page, click the three dots icon for the cluster you want to pause and choose **Pause Cluster**. You can also select the cluster, click **More Links**, and click **Pause Cluster**.
+
+Yugabyte notifies you when a cluster is paused and resumed.
```

Comment:
An email is sent only when cluster is in paused state for 30 days (more like a gentle nudge to remind them it can be resumed). No email sent when resumed.